test/e2e/apps/disruption.go

```diff
@@ -247,6 +265,14 @@ func createPDBMaxUnavailableOrDie(cs kubernetes.Interface, ns string, maxUnavail
 	Expect(err).NotTo(HaveOccurred())
 }
 
+func updatePDBMinAvailableOrDie(cs kubernetes.Interface, ns string, minAvailable intstr.IntOrString) {
+	pdb, err := cs.PolicyV1beta1().PodDisruptionBudgets(ns).Get("foo", metav1.GetOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	pdb.Spec.MinAvailable = &minAvailable
+	_, err = cs.PolicyV1beta1().PodDisruptionBudgets(ns).Update(pdb)
```

Comment:
This could make the test flaky, since the Update can fail due to a ResourceVersion error if it races with other updates such as the controller updating status (we use optimistic concurrency).

Here's an example of using the RetryOnConflict helper to avoid this:

https://sourcegraph.com/github.com/kubernetes/kubernetes@7693a1d/-/blob/test/e2e/framework/service_util.go#L1071-1089